src/components/Player/Pages/Overview/Overview.css

```diff
@@ -1,19 +1,17 @@
 @import "../../../palette.css";
 
-.overviewContainer {
-  display: flex;
+.overviewMatches, .overviewHeroes {
+  display: inline-block;
+  @media only screen and (max-width: 1180px) {
+    width: 100% !important;
+    margin: 0;
+  }
 }
 
 .overviewMatches {
-  width: 75%;
-  display: inline-block;
+  width: calc(70% - 30px);
+  margin-right: 30px;
```

Comment:
I think we should use a media query to make the tables stack at smaller screen sizes (tablet landscape or tablet portrait?)